util/__init__.py

```diff
@@ -27,6 +27,7 @@
 CKAN_TEST_SQLALCHEMY_URL = os.environ.get("CKAN_TEST_SQLALCHEMY_URL", "postgresql://ckan_default:pass@db/ckan_test")
 ADMIN_APIKEY = os.environ.get("ADMIN_APIKEY", "6011357f-a7f8-4367-a47d-8c2ab8059520")
 CKAN_SITE_URL = os.environ.get("CKAN_SITE_URL", "http://adr.local")
+SKIP_DB_RESTART = (os.environ.get("SKIP_DB_RESTART", False) == 'True')
```

Comment:
tak should look like this I think:
```
SKIP_DB_RESTART = (os.environ.get("SKIP_DB_RESTART", 'false').lower() == 'true')
```
your example is not going to work for
`export SKIP_DB_RESTART=true`

Reply:
You're right, but it's also not going to work for "1", "yes" or "OK". In Python True is boolean but "true" is just a string, so this was deliberate, but sure, I can make it accept "true" too. 

I don't get it,  why do you suggest I should change False to 'false' ?

Reply:
Thanks for the explanation, fixed